nuitka/build/static_src/CompiledCodeHelpers.c

```diff
@@ -541,6 +541,43 @@ PyObject *BUILTIN_LEN(PyObject *value) {
     return PyInt_FromSsize_t(res);
 }
 
+PyObject *BUILTIN_ANY(PyObject *value) {
+    CHECK_OBJECT(value);
+
+    PyObject *it = PyObject_GetIter(value);
+
+    if (unlikely((it == NULL))) {
+    	return NULL;
+    }
+
+
+    int cmp;
```

Comment:
The `cmp` is only used inside the loop, move this declaration it to its assigment